### PR TITLE
ui: Fix column names translation issue

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR fixes the issue in displaying column names in the dropdown menu
error1
issue2

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### How Has This Been Tested?
fix1
fix2

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan ui

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -444,9 +444,9 @@
 "label.annotated.by": "Annotator",
 "label.annotation": "Comment",
 "label.annotations": "Comments",
-"label.annotation.admins.only": "Only visible to Administrators",
```
Comment (nvazquez) [inline ui/public/locales/en.json:447]: Please check the usage of these keys - they break the comments feature UI:
Screen Shot 2022-02-22 at 11 32 41

Comment (Pearl1594) [reply ui/public/locales/en.json:447]: Thanks @nvazquez , I've updated it.

**Comment by nvazquez** [comment]
@blueorangutan ui

File: ui/public/locales/en.json (hunk 1 above)
Comment (nvazquez) [reply ui/public/locales/en.json:447]: Thanks

**Review by nvazquez — APPROVED** [review]
Thanks @Pearl1594 LGTM - tested OK

**Review by GabrielBrascher — APPROVED** [review]
LGTM

**Review by GutoVeronezi — APPROVED** [review]
CLGTM

**Review by utchoang — APPROVED** [review]

**Pull request merged**